src/wheel/_bdist_wheel.py

```diff
@@ -34,6 +34,15 @@
 if TYPE_CHECKING:
     import types
 
+# ensure Python logging is configured
+try:
+    __import__("setuptools.logging")
```

Comment:
```suggestion
    import setuptools.logging
```

Why not this? They should be the same, right? Usually you use this form if you have a variable value to import.

Reply:
(Though now I see this is just moved...)